release-notes/schemas/dotnet-os-packages.json

```diff
@@ -0,0 +1,184 @@
+{
+  "description": "The set of packages required by a given product version for a set of distros.",
+  "type": [
+    "object",
+    "null"
+  ],
+  "properties": {
+    "channel-version": {
+      "description": "Major (or major.minor) version of product.",
+      "type": "string"
+    },
+    "packages": {
+      "description": "Set of nominal packages used by product, with descriptions.",
+      "type": "array",
+      "items": {
+        "description": "A nominal package is a distro-agnostic representation of a package, including the scenarios for which the package is required. A nominal package will be referenced by a distribution package, with a distribution-specific package name.",
+        "type": [
+          "object",
+          "null"
+        ],
+        "properties": {
+          "id": {
+            "description": "ID of nominal package.",
+            "type": "string"
+          },
+          "name": {
+            "description": "Display name of nominal package.",
+            "type": "string"
+          },
+          "required-scenarios": {
+            "description": "Required scenarios for which the package must be used.",
+            "type": "array",
+            "items": {
+              "description": "Scenarios relating to package dependencies. \u0027All\u0027 includes both CoreCLR and NativeAOT while \u0027Runtime\u0027 is intended to cover CoreCLR, only.",
+              "enum": [
+                "all",
+                "runtime",
+                "https",
+                "cryptography",
+                "globalization",
+                "kerberos"
+              ]
+            }
+          },
+          "references": {
+            "description": "Related references.",
+            "type": [
+              "array",
+              "null"
+            ],
+            "items": {
+              "type": [
+                "string",
+                "null"
+              ]
+            },
+            "default": null
+          }
+        },
+        "required": [
+          "id",
+          "name",
+          "required-scenarios"
+        ]
+      }
+    },
+    "distributions": {
+      "description": "Set of distributions where the product can be used.",
+      "type": "array",
+      "items": {
+        "description": "An operating system distribution, with required package install commands and specific packages for distribution releases.",
+        "type": [
+          "object",
+          "null"
+        ],
+        "properties": {
+          "name": {
+            "description": "Name of the distribution, matching ID in /etc/os-release,however, the expectation is that this value starts with a capital letter (proper noun).",
```

Comment:
```suggestion
            "description": "Name of the distribution, matching ID in /etc/os-release, however, the expectation is that this value starts with a capital letter (proper noun).",
```

Reply:
Thanks! I updated the object model with this change and then re-generated the JSON schema.